packages/cf-component-icon/test/__snapshots__/Icon.js.snap

```diff
@@ -1,54 +1,37 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`should render role 1`] = `
-<i
-  className="cf-icon cf-icon--loading"
-  role="status"
-/>
-`;
-
 exports[`should render type 1`] = `
 <i
-  className="cf-icon cf-icon--info-sign"
-  role={undefined}
+  className="cf-1ew9cp5"
+  id={undefined}
```

Comment:
I made a comment about it elsewhere in this PR. Seems to come from `createComponent`. I'm at least not adding these props anywhere.